horizons/ai/aiplayer/productionbuilder.py

```diff
@@ -263,24 +263,24 @@ def display(self):
 			return
 
 		tile_colors = {
-			BUILDING_PURPOSE.ROAD:            ( 30,  30,  30),
```

Comment:
So what would you suggest?

To find out what is and isn't possible, we have to check the code that's actually using these numbers. Which line(s) would that be here?